src/assignment/components/AssignmentActions.tsx

```diff
@@ -71,7 +70,12 @@ const AssignmentActions: FunctionComponent<AssignmentActionsProps> = ({
 				config?.button?.className
 			)}
 		>
-			<ShareAssignmentWithPupil {...share} {...config} />
+			<ShareDropdown
+				users={mockShareUsers}
+				onAddNewUser={(value: Partial<ShareUserInfo>) => console.log(value)}
+				onDeleteUser={(value) => console.log(value)}
+				onEditRights={(user, newRights) => console.log(user, newRights)}
+			/>
```

Comment:
mockdata and console.log's will be replaced with actual data, right?

Reply:
Yes